build/dependencies.props

```diff
@@ -2,23 +2,29 @@
   <PropertyGroup>
     <AspNetCoreVersion>2.0.0-*</AspNetCoreVersion>
     <CoreFxVersion>4.3.0</CoreFxVersion>
-    <DapperVersion>1.*</DapperVersion>
     <InternalAspNetCoreSdkVersion>2.1.0-*</InternalAspNetCoreSdkVersion>
     <JsonNetVersion>10.0.1</JsonNetVersion>
     <NETStandardImplicitPackageVersion>2.0.0-*</NETStandardImplicitPackageVersion>
-
-    <!--
-      * Once Npgsql 3.2.0 or later is stable, the required version should be changed to "3.*".
-      * For now, we need to use "3.2.0-*" since there's no way to float the latest pre-release version.
-    -->
-    <NpgsqlVersion>3.2.0-*</NpgsqlVersion>
-    <NpgsqlEntityFrameworkCorePostgreSQLVersion>1.*</NpgsqlEntityFrameworkCorePostgreSQLVersion>
+    <NETStandardLibraryNETFrameworkVersion>2.0.0-*</NETStandardLibraryNETFrameworkVersion>
+    <RuntimeFrameworkVersion Condition="'$(TargetFramework)'=='netcoreapp2.0'">2.0.0-*</RuntimeFrameworkVersion>
 
     <!--
       * Use 4.4.0-* instead of $(CoreFxVersion) to address "SqlClient fails with netcoreapp2.0 on Win7/Server2008"
         * https://github.com/dotnet/corefx/issues/18406
       * Revert if and when $(CoreFxVersion) is upgraded to 4.4.0-*
     -->
     <SqlClientVersion>4.4.0-*</SqlClientVersion>
+
+    <!--
+      Following properties are specific to src/Benchmarks/Benchmarks.csproj.
```

Comment:
So that we can test previous releases without having to undo tracking changes in all of the web projects. This repo is far easier to automate than our fork of the TechEmpower repo.

I plan to port my branches for the existing releases from that fork here.